### ISSUE TITLE
Hook up "Show Comparison" button

### ISSUE THREAD

**Issue by justinanderson** [CxSci/SA-dashboard#105]
This depends on functionality in #104, so work on this should branch from locations-to-compare and then rebase onto master after locations-to-compare has been merged into master.

We've just about got everything leading up to comparison mode implemented, now we need to make the ["Show Comparison" button](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1517%3A15316) lead to [comparison mode](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1517%3A15356).

This task is very basic. When a user clicks on "Show Comparison", we should:
- [x] Replace the sidebar search field / location name with a ["<- Comparing Locations" title and back button](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1489%3A464) which exits comparison mode.
- [x] Replace the map with a big white div.
- [x] Update the hamburger menu's appearance to match its [Light background](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1446%3A10019) style.

So the end result of this issue should look like [this Figma frame](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1872%3A22766).

There are several other tasks after this one is done which can be done in parallel, so let's keep this one simple.

**Comment by justinanderson** [comment]
Resolved by #107.
